src/components/Map/LayerControl.tsx

```diff
@@ -42,8 +42,8 @@ export const LayerControl = ({ locations, onMarkerClick }: Props) => {
           return (
             <HeatmapLayer
               key={idx}
-              fitBoundsOnUpdate
-              fitBoundsOnLoad
+              fitBoundsOnUpdate={false}
+              fitBoundsOnLoad={false}
```

Comment:
This line was causing map to zoom out to fit all of the heatmap onto the viewport.